plugins/ompgsql/ompgsql.c

```diff
@@ -255,7 +255,8 @@ tryExec(uchar *pszCmd, wrkrInstanceData_t *pWrkrData)
 	pgRet = PQexec(pWrkrData->f_hpgsql, (char*)pszCmd);
 	execState = PQresultStatus(pgRet);
 	if (execState != PGRES_COMMAND_OK && execState != PGRES_TUPLES_OK) {
-		dbgprintf("postgres query execution failed: %s\n", PQresStatus(PQresultStatus(pgRet)));
+	// complain a lot in case any issues with DB communication
```

Comment:
proper indention would be great

Reply:
Done. (artifact of my editor messing tab and spaces)